rules/windows/builtin/win_mmc20_lateral_movement.yml

```diff
@@ -16,9 +16,9 @@ logsource:
     product: windows
 detection:
     selection:
-        ParentImage: '*\svchost.exe'
-        Image: '*\mmc.exe'
-        CommandLine: '*-Embedding*'
+        ParentImage|endswith: '\svchost.exe'
+        Image|endswith: '\mmc.exe'
+        CommandLine|endswith: '-Embedding*'
```

Comment:
should be `CommandLine|contains: '-Embedding'` here (:

Reply:
Fixed :)

Reply:
Fixed :)